src/types/template.paths.ts

```diff
@@ -4,6 +4,7 @@ export enum Templates {
     ERROR_404 = "page-not-found",
     SERVICE_OFFLINE = "service-offline",
     SERVICE_OFFLINE_MID_JOURNEY = "service-offline-mid-journey",
+    SIC = "confirmation-statement/sic",
```

Comment:
I think Matt has created a subfolder called 'tasks' for the task list  pages, could you move yours into there? perhaps even create a sic subfolder under tasks if you think that might be clearer